config/itemTypes.json

```diff
@@ -16,6 +16,9 @@
 }, {
   "id": "/Archwing/Melee",
   "name": "Arch-Melee"
+}, {
+  "id": "/Sigils",
+  "name": "Sigil"
```

Comment:
this would set the type, not the category

Reply:
i'm down with the change, but it's not going to categorize, just change the type (which would probably keep it in Skins.json)

Reply:
lol, here's how dumb i am. this would totally get the job done.

https://github.com/WFCD/warframe-items/blob/ce433f807af546fa518042bf4fb20a34f8acf6b9/build/parser.mjs#L567-L569 (thanks to this)